R/odbc_dbFetch.R

```diff
@@ -1,11 +1,13 @@
 #' Fetch query result from database
 #'
-#' This method is an adaptation from the eponymous function in the `odbc` package and is an implementation of the method `dbFetch` defined in the `DBI` package.  Additional to the `odbc` package, it replaces an cryptic error message by an informative error message.  In Windows OS, it moreover assures that characters are interpreted correctly (while their presentation depends on the settings of the locale).
+#' This method is an adaptation from the eponymous function in the `odbc` package and is an implementation of the method `dbFetch` defined in the `DBI` package.  Additional to the `odbc` package, it replaces an cryptic error message by an informative error message.  In Windows OS and if chosen encoding = "utf-8" in connect_inbo_dbase(), it moreover assures that characters are interpreted correctly (while their presentation depends on the settings of the locale).
```

Comment:
```suggestion
#' This method is an adaptation from the eponymous function in the `odbc` package and is an implementation of the method `dbFetch` defined in the `DBI` package.  Additional to the `odbc` package, it replaces an cryptic error message by an informative error message.  In Windows OS and if chosen `autoconvert_utf8 = TRUE` in connect_inbo_dbase(), it moreover assures that characters are interpreted correctly (while their presentation depends on the settings of the locale).
```

Also Rd file is to be updated if you commit this.